frontend/public/style/_common.scss

```diff
@@ -105,6 +108,11 @@ $co-external-link-padding-right: 15px;
   }
 }
 
+.co-m-pane__heading-badge {
+  margin-left: var(--pf-global--spacer--sm);
```

Comment:
Nit:  this declaration should come second (move to line 113) as we aim to keep declarations alphabetized by property.